apps/meteor/app/importer/server/classes/ImportDataConverter.ts

```diff
@@ -303,6 +303,8 @@ export class ImportDataConverter {
 		}
 		await this.updateUser(user, userData);
 
+		Object.assign(user, userData);
```

Comment:
Those two objects do not have the same type for all attributes. Instead of assigning the data directly here, you should load the user from the db again.